actions/file_control_MOD.js

```diff
@@ -143,6 +212,7 @@ class FileControl {
     const dirName = path.normalize(this.evalMessage(data.filepath, cache))
     const fileName = this.evalMessage(data.filename, cache)
     const line = parseInt(this.evalMessage(data.input2, cache))
+    const togglestat = this.evalMessage(data.togglestatus.checked, cache)
```

Comment:
For `data.togglestatus` to be a valid data you need to add a string `togglestatus` into the `fields` array.

Reply:
Additionally, this.evalMessage is used to evaluate strings, simply leave data.togglestatus, without .checked

Reply:
.checked is required to catch the boolean